test/DebugInfo/transferring.swift

```diff
@@ -0,0 +1,24 @@
+// RUN: %target-swift-frontend -emit-ir -g -o /dev/null -module-name test -strict-concurrency=complete -swift-version 5 -enable-experimental-feature TransferringArgsAndResults -disable-availability-checking %s
+
```

Comment:
can you add a CHECK: that ensures that the mangled name you're looking for appears in a DI*Type?